Packages/com.reese.nav/Agent/NavGroundingSystem.cs

```diff
@@ -55,12 +53,12 @@ protected override void OnUpdate()
                        agent.SurfacePointNormal = hit.SurfaceNormal;
 
                        var currentPosition = translation.Value;
-                       currentPosition.y = hit.Position.y + agent.Offset.y;
+                       currentPosition.y = hit.Position.y;
```

Comment:
Should not be setting the Agent's physical position by the offset. I think I misunderstood what the offset was for.